cmd/api/acrsdk.go

```diff
@@ -21,10 +21,21 @@ import (
 
 // Constants that are used throughout this file.
 const (
-	prefixHTTPS           = "https://"
-	registryURL           = ".azurecr.io"
-	manifestTagFetchCount = 100
-	manifestV2ContentType = "application/vnd.docker.distribution.manifest.v2+json"
+	prefixHTTPS                      = "https://"
+	registryURL                      = ".azurecr.io"
+	manifestTagFetchCount            = 100
+	manifestORASArtifactContentType  = "application/vnd.cncf.oras.artifact.manifest.v1+json"
+	manifestOCIArtifactContentType   = "application/vnd.oci.artifact.manifest.v1+json"
+	manifestOCIImageContentType      = "application/vnd.oci.image.manifest.v1+json"
+	manifestOCIImageIndexContentType = "application/vnd.oci.image.index.v1+json"
```

Comment:
Should we reuse the media types defined in https://github.com/opencontainers/image-spec/blob/main/specs-go/v1/mediatype.go

Reply:
We did not do that for docker media types because we don't want to introduce the dependency on distribution.

Reply:
updated